plugins/languages/javascript.vim

```diff
@@ -2,9 +2,7 @@ Plug 'pangloss/vim-javascript', { 'for': 'javascript' }
 
 Plug 'mxw/vim-jsx', { 'for': 'javascript' }
 
-Plug 'leshill/vim-json', { 'for': 'javascript' }
-
 Plug 'ternjs/tern_for_vim', { 'for': 'javascript' }
 
-Plug 'maksimr/vim-jsbeautify', { 'for': 'javascript' }
+Plug 'maksimr/vim-jsbeautify'
```

Comment:
Because it created an error when you saved an html, json, css file. It called jsbeautify, but it was not loaded.